.github/workflows/release.yaml

```diff
@@ -116,9 +116,11 @@ jobs:
 
     - name: 'Identify New Contributors Section'
       id: get-new-contributors
+      env:
+        body: "${{ fromJson(steps.get-github-release-notes.outputs.data).body }}"
```

Comment:
Does this set the “$body” variable? Because below we use a new variable called “$data” instead

Reply:
Ouch. Thanks for catching. Updated.